config/windows.json

```diff
@@ -0,0 +1,513 @@
+{
+	"randomMapTab":
+	{
+		"items":
+		[
```

Comment:
Since widget names must be non-empty, unique string perhaps replace "items" array with struct/object, with widget name acting as key?
```
"background : { "type" : "...", "image" : "..." }
```

Reply:
you may want to control draw order, but only json array guarantees the sequential iterating